lib/data/multipleBackendGateway.js

```diff
@@ -273,6 +273,14 @@ const multipleBackendGateway = {
           'Can not copy object from ' +
           `${client.clientType} to ${client.clientType}`));
     },
+    protectAzureBlocks: (bucketName, objectKey, location, log, cb) => {
```

Comment:
let's keep this in the azureclient itself and call it at the beginning of put and delete in the client itself.

Reply:
as discussed, this is a sad state of affairs but we need it outside to avoid orphans.